drivers/hmis/app/graphql/types/hmis_schema/enums/project_config_type.rb

```diff
@@ -13,6 +13,7 @@ class HmisSchema::Enums::ProjectConfigType < Types::BaseEnum
     value 'AUTO_EXIT', description: 'Auto Exit'
     value 'AUTO_ENTER', description: 'Auto Enter'
     value 'STAFF_ASSIGNMENT', description: 'Staff Assignment'
-    value 'COORDINATED_ENTRY', description: 'Coordinated Entry'
+    value 'COORDINATED_ENTRY', description: 'Supports Coordinated Entry Referrals'
+    value 'SENDS_DIRECT_CE_REFERRALS', description: 'Sends Direct Coordinated Entry Referrals'
```

Comment:
The naming convention doesn't match other config types, for example we don't say "Supports Auto Exit" or "Supports Staff Assignment". We probably do need some more descriptive explainers for all of these, but I think for now I would prefer to match the existing convention of just naming the config type without adding a verb. Maybe "Coordinated Entry Referrals" and "Outgoing Referrals"?